packages/l10n/src/create-l10n-injector/create-l10n-injector.tsx

```diff
@@ -98,3 +113,51 @@ export function createL10NInjector<LoadedData>({
     };
   };
 }
+
+export function createLegacyL10NInjector<LoadedData extends {}>({
```

Comment:
This is basically the "old" implementation

Reply:
Hm. Looks good to me. I personally think we're trying to jump onto the hook train a bit too eagerly given this but that's just my humble view.